src/Symfony/Component/Cache/Traits/PhpArrayTrait.php

```diff
@@ -65,36 +68,25 @@ public function warmUp(array $values)
 
 EOF;
 
+        $exportSerializer = new PhpExportSerializer();
```

Comment:
PhpExportSerializer is less capable than the current code, which accepts all serializable objects.
We should preserve this property.

Reply:
I can merge PhpExportSerializer into PhpArrayTrait. It is not used in other Caches and it is not part of serialization contract of PhpArrayTrait.